### ISSUE TITLE
Publish what we have?

### ISSUE THREAD

**Issue by mfandl** [Peculiar-Computer-Club/pcc-wiki#5]
I think the present state with what @fanna already did with C section is already valuable for the outside world. I suggest deleting all the TODO stuff, publish what we have and graudually add things along the way.

Suggestions on where to publish are welcome. I am OK with github pages, but also with buying our own hosting + domain. 

What do you think?

**Comment by fanna** [comment]
I guess Github pages are sufficient for hosting, and we can just buy some domain and namespace it. 

As for TODOs - we should probably move them to some text file or just set some reminders so we don't forget about it.

**Comment by mfandl** [comment]
True, I'll move them to a text file. We could also open issues for them later as @yanchith suggested. I'm assigning this issue to myself.

Also, sorry for mixing the issues, but this is technically not a wiki. See [this](https://en.wikipedia.org/wiki/Wiki) and [this](https://en.wikipedia.org/wiki/Wiki_software). Shall we rename this to something else? _pcc knowledge base_ or _pcc grimoire_?

**Comment by yanchith** [comment]
Grimoire ❤️
